src/styles/navigation.scss

```diff
@@ -0,0 +1,55 @@
+.navigation {
+  margin-right: 50px;
```

Comment:
It's better to use `min-width` + `margin: auto` to center header and give for it spacing from corners.